Modules/Core/Common/include/itkSymmetricEigenAnalysis.hxx

```diff
@@ -36,16 +36,6 @@ SymmetricEigenAnalysis< TMatrix, TVector, TEigenMatrix >::ComputeEigenValues(con
     {
     return ComputeEigenValuesLegacy(A, D);
     }
-
-#ifndef NDEBUG
```

Comment:
Does this block produce a warning?

Reply:
For Debug configurations, yes, it does (when compiling with MSVC warning level 4). And indeed, the code is unreachable, even in debug-mode, because the function already _returns_ before that block.